From c24810a0db9814e999da43bbbe0412db28a8282a Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Tue, 26 Jun 2012 17:57:58 +0800
Subject: [PATCH 1698/2508] ENGR00213014-7 MXC EDID: Export function
 mxc_edid_parse_ext_blk

Export function mxc_edid_parse_ext_blk.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 drivers/video/mxc/mxc_edid.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/video/mxc/mxc_edid.c b/drivers/video/mxc/mxc_edid.c
index 6aec72d..c039992 100644
--- a/drivers/video/mxc/mxc_edid.c
+++ b/drivers/video/mxc/mxc_edid.c
@@ -412,6 +412,7 @@ int mxc_edid_parse_ext_blk(unsigned char *edid,
 
 	return 0;
 }
+EXPORT_SYMBOL(mxc_edid_parse_ext_blk);
 
 static int mxc_edid_readblk(struct i2c_adapter *adp,
 		unsigned short addr, unsigned char *edid)
-- 
1.7.7.4

